edx_django_utils/monitoring/docs/how_tos/add_code_owner_custom_metric_to_an_ida.rst

```diff
@@ -0,0 +1,46 @@
+Add Code_Owner Custom Metric to an IDA
+======================================
+
+.. contents::
+   :local:
+   :depth: 2
+
+What is the code_owner custom metric?
+-------------------------------------
+
+The code_owner custom metric can be used to create custom dashboards and alerts for monitoring the things that you own. It was originally introduced for the LMS, as is described in this `ADR on monitoring by code owner`_.
+
+You can now easily add this same metric to any IDA so that your dashboards and alerts can work across multiple IDAs at once.
+
+If you want to know about custom metrics in general, see: using_custom_metrics.rst.
+
+.. _ADR on monitoring by code owner: https://github.com/edx/edx-platform/blob/59e0f6efcf2a297806918f8e0020255c1f59ea5f/lms/djangoapps/monitoring/docs/decisions/0001-monitoring-by-code-owner.rst
+
+Setting up the Middleware
+-------------------------
+
+You simply need to add ``edx_django_utils.monitoring.code_owner.middleware.CodeOwnerMetricMiddleware`` as described in the README to make this functionality available. Then it is ready to be configured.
+
+Configuring your app settings
+-----------------------------
+
+Once the Middleware is made available, simply set the Django Setting ``CODE_OWNER_MAPPINGS`` appropriately.
+
+The following example shows how you can include an optional config for a catch-all using ``'*'``.  The example is provided in YAML format because quotes are required to not break the YAML.
```

Comment:
I'm a little unclear on this part:

> The example is provided in YAML format because quotes are required to not break the YAML.

Is this saying that you could have illustrated this in other formats, but that you chose YAML because it requires special care and therefore benefits the most from use in the documentation? (That makes sense to me, but the explanation is a little opaque as written.)

Reply:
I tried to improve.  Please let me know if you like it, and if not, please provide and alternative that you like.  Thanks!

Reply:
Just saw your updated thumb.  Thanks.